website/project/model.py

```diff
@@ -890,6 +882,8 @@ def institution_relationship_url(self):
 
     def __init__(self, *args, **kwargs):
 
+        kwargs.pop('logs', [])
```

Comment:
If I understood correctly, I was told to remove logs as a field and make it a property, but not have the migration clear it out of the database.  So old projects before the migration would still have log data in the background.

When __init__ was called on an old project, the logs data would still be there and it would fail here.  Popping off the logs allowed me to get around it and still leave old log data in the db.